apps/website/docs/pipelines/azure-login.md

```diff
@@ -5,35 +5,72 @@ sidebar_label: Configuring Azure Login for GitHub Actions
 
 # Configuring Azure Login for GitHub Actions
 
-There are several ways to log into Azure by a GitHub Action. The DX chosen
-method doesn't require maintenance or secret management as it is a passwordless
-approach. Once set up, it just works.
+There are several ways to log into Azure using a GitHub Action. The DX-preferred
+method is passwordless, requiring no maintenance or secret management. Once set
+up, it works seamlessly.
 
-Only two components are needed to let GitHub Actions log into Azure:
+:::tip
+
+The modules `azure_federated_identity_with_github` and
+`azure_github_environment_bootstrap` already implement the steps described in
```

Comment:
suggestion: "already implement the steps described in this article" is ambiguos. suppose the reader wants to use DX modules, which parts of this guide can he skip? I'd be more explicit about this, eventually splitting the guide in two parts, one for modules users and the other one for naive ones.

Reply:
Added some guidance